From 89c7fc15e064dd407e6d5ccefe78985b86b8a036 Mon Sep 17 00:00:00 2001
From: Brendan Heading <brendanheading@gmail.com>
Date: Tue, 1 Sep 2015 12:26:36 +0100
Subject: [PATCH 1/1] fix compilation under musl

Patch borrowed from Void Linux :

https://github.com/voidlinux/void-packages/blob/master/srcpkgs/dmraid/patches/25_musl-libc.patch

Upstream package appears dormant, no mailing list and no
maintainence releases since Nov 2010.

Upstream-Status: dormant
Signed-off-by: Brendan Heading <brendanheading@gmail.com>

---
 1.0.0.rc16-3/dmraid/include/dmraid/misc.h | 1 +
 1.0.0.rc16-3/dmraid/lib/device/scan.c     | 1 +
 2 files changed, 2 insertions(+)

diff --git a/1.0.0.rc16-3/dmraid/include/dmraid/misc.h b/1.0.0.rc16-3/dmraid/include/dmraid/misc.h
index 247a68e..1b1dfdf 100644
--- a/1.0.0.rc16-3/dmraid/include/dmraid/misc.h
+++ b/1.0.0.rc16-3/dmraid/include/dmraid/misc.h
@@ -10,6 +10,7 @@
 
 #ifndef _MISC_H_
 #define _MISC_H_
+#include <fcntl.h>
 
 #define DM_ASSERT(__cond) do { if (!(__cond)) { printf("ASSERT file:%s line:%d fuction:%s cond: %s\n", __FILE__, __LINE__, __FUNCTION__, #__cond); } } while(0);
 
diff --git a/1.0.0.rc16-3/dmraid/lib/device/scan.c b/1.0.0.rc16-3/dmraid/lib/device/scan.c
index a040d53..6897146 100644
--- a/1.0.0.rc16-3/dmraid/lib/device/scan.c
+++ b/1.0.0.rc16-3/dmraid/lib/device/scan.c
@@ -10,6 +10,7 @@
 # include <dirent.h>
 # include <paths.h>
 #else
+# include <paths.h>
 # include <dirent.h>
 # include <mntent.h>
 #endif
-- 
2.4.3

